schemas/Account.json

```diff
@@ -39,6 +39,10 @@
     "ledger": {
       "description": "A link the the account's ledger",
       "type": "Iri.json"
+    },
+    "fingerprint": {
+      "description": "A fingerprint of the account's client certificate",
+      "type": "string"
```

Comment:
Definitely agree. One note: The length of the fingerprint depends on the hashing function used. Right now the longest hashing function that's in active use would be SHA-512, which would be `^[0-9A-F]{2}(:[0-9A-F]{2}){,63}$`. We could also go to 1024 bits to have some future-proofing.

Also, should it be case-insensitive? For instance, the Wikipedia example for a fingerprint is lowercase.